libs/utils/trace.py

```diff
@@ -230,8 +215,13 @@ def __parseTrace(self, path, window, trace_format):
         else:
             raise ValueError("Unknown trace format {}".format(trace_format))
 
-        self.ftrace = trace_class(path, scope="custom", events=self.events,
-                                  window=window, normalize_time=self.normalize_time)
+        if self.normalize_time:
+            self.ftrace = trace_class(path, scope="custom", events=self.events,
+                                      window=window, normalize_time=self.normalize_time)
+        else:
+            self.ftrace = trace_class(path, scope="custom", events=self.events,
+                                      abs_window=window, normalize_time=self.normalize_time)
```

Comment:
Maybe instead of having two `trace_class` call sites you could have something like
```
# If using normalized time, we need to use TRAPpy's `abs_window` param instead of `window`
if self.normalize_time:
    abs_window = window
    window = (0, None)
else:
    abs_window = (0, None)
    window = window

self.ftrace = trace_class(...)
```
It's actually longer but perhaps clearer? Up to you.

Reply:
While writing this I was thinking we could maybe use a dictionary to describe the args and feed it unpacked to the `trace_class` call - would that be a good practice ?

Reply:
Yeah I thought of that too, but it feels a bit runic, and anyway I think it would end up looking just like the code in my example, right? But yeah it's just style so up to you really. Can keep the current code too if you prefer.